bootstrap/src/main/scala/org/scalafmt/bootstrap/Bootstrap.scala

```diff
@@ -77,7 +77,7 @@ object ScalafmtBootstrap {
         logger = Some(logger)
       )
     )
-    val resolution = start.process.run(fetch).unsafePerformSync
```

Comment:
That's not too big of an issue IMO, it's OK to classload a few instances of scalafmt. sbt does a whole lot more of redundant classloading :D